lua/rip-substitute/config.lua

```diff
@@ -76,9 +85,9 @@ function M.setup(userConfig)
 
 	-- DEPRECATION
 	if M.config.incrementalPreview.hlGroups then
-		local msg =
-			"`incrementalPreview.hlGroups.{name}` is deprecated, use `incrementalPreview.matchHlGroup` instead."
-		notify(msg, "warn")
+		-- local msg =
```

Comment:
as annoying as it is, I'm afraid we will need a deprecation notice for the now basically reverted setting…

Reply:
just to make sure I understand correctly:
the new deprecation notice would be:
	"`incrementalPreview.matchHlGroup` is deprecated, use `incrementalPreview.hlGroups.{name}` instead."
	
	?

Reply:
yep